streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -216,7 +230,7 @@ public void maybeAddMetric(final Sensor sensor, final MetricName name, final Mea
      * @param action Action to run.
      * @param sensor Sensor to record value.
      */
-    void measureLatencyNs(final Time time, final Runnable action, final Sensor sensor) {
+    public void measureLatencyNs(final Time time, final Runnable action, final Sensor sensor) {
```

Comment:
Because I moved StreamsMetricsImpl to the `...internal.metrics` package, but I took a look, and it was only used (properly) by `ProcessorNode`.

`StreamTask` also used it, but only by specifically wrapping the operation in a `Runnable` and passing it to the metric to immediately be run.

I've added https://github.com/apache/kafka/pull/4812/commits/18358f30445cd27c5155be7d73b8d5a98410886a to simplify `StreamTask`'s usage to not need this method, and move the method to `ProcessorNode`.